wait-for-httpbin.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# httpbin sometimes times out, but comes back shortly after this
+# happens. Apply a basic backoff scheme to give a chance to the tests
+# to run later instead of failing.
+for i in 1 2 3
+do
+    if curl http://httpbin.org >/dev/null
+    then
+	break
```

Comment:
Would be nice if you can change this to spaces instead of a tab, but it's not essential.